config.frontend.test.yaml

```diff
@@ -29,6 +29,8 @@ default_project: &default_project
                     host: https://citoid-beta.wmflabs.org
                   recommendation:
                     host: https://recommendation-api-beta.wmflabs.org
+                  appeditortasks:
+                    host: https://app-editor-tasks.wmflabs.org
```

Comment:
What is this domain pointing too? it's a different instance of recommendation api service or what is this?

Reply:
That's the labs instance @mdholloway was working on. I think this instance has a special setup for the endpoints to work with [Extension:WikimediaEditorTasks](https://www.mediawiki.org/wiki/Extension:WikimediaEditorTasks), commons and wikidata.

Reply:
Hm. but where would we point production too? To recommendation-api LVS? I think we need a bit more info from @mdholloway here.

Reply:
@Pchelolo see, https://github.com/wikimedia/restbase/pull/1153#issuecomment-507436692

Reply:
> Hm. but where would we point production too? To recommendation-api LVS? I think we need a bit more info from @mdholloway here.

Yes, the deployment followed the regular recommendation-api target machines. The limitations of the beta-cluster that made us create a new instance to perform the actions the Apps team needed for testing.

Reply:
I'm not entirely sure why the normal `recommendation-api` service can not handle these recommendations.. Since the normal recommendation-api.beta.wmflabs.org is as well completely under our control, why can't we make that one work and not create additional installations of the same thing?

Reply:
Perhas cause the recommendation service installed in beta doesn't seem to work. Then we need to fix it and not create a separate installation.

Reply:
This should indeed point to the Beta Cluster. `app-editor-tasks` was just for prototyping and testing, particularly before the code was merged onto recommendation-api master.

Reply:
(Pointing to `app-editor-tasks` in this PR rather than BC like the others was probably just an error on my part.)

Reply:
Ah, OK, I see what's going on here.  Yeah, we definitely don't want to introduce an `appeditortasks` service to RB.  I'll update the PR.